packages/react/src/components/MultiSelect/__tests__/MultiSelect-test.js

```diff
@@ -528,4 +530,84 @@ describe('MultiSelect', () => {
       );
     });
   });
+
+  describe('Controlled', () => {
+    const ControlledMultiselect = () => {
+      const items = generateItems(4, generateGenericItem);
+      const [selectedItems, setSelectedItems] = useState([]);
+
+      const onSelectionChanged = (value) => {
+        setSelectedItems(value);
+      };
+      return (
+        <>
+          <MultiSelect
+            id="test"
+            titleText="Multiselect title"
+            label="test-label"
+            items={items}
+            selectedItems={selectedItems}
+            onChange={(data) => onSelectionChanged(data.selectedItems)}
+            selectionFeedback="top-after-reopen"
+          />
+          <br />
+          <ButtonSet>
+            <Button
+              id="all"
+              onClick={() =>
+                setSelectedItems(items.filter((item) => !item.disabled))
+              }>
+              Select all
+            </Button>
+            <Button
+              id="clear"
+              kind="secondary"
+              onClick={() => setSelectedItems([])}>
+              Clear
+            </Button>
+          </ButtonSet>
+        </>
+      );
+    };
+
+    it('should initially render controlled multiselect with a given label', () => {
```

Comment:
Had put this in the beginning to make sure that the `ControlledMultiSelect` actually renders if we have passed items through `selectedItems`. Just like we have one of this test in the beginning of `MultiSelect`, just to make sure if anyone made changes to how `selectedItems` work, then this could potentially catch the test.